test/Microsoft.AspNet.Mvc.TagHelpers.Test/SelectTagHelperTest.cs

```diff
@@ -243,12 +243,9 @@ public class SelectTagHelperTest
             Assert.Equal(expectedTagName, output.TagName);
 
             Assert.NotNull(viewContext.FormContext?.FormData);
-            var keyValuePair = Assert.Single(
+            Assert.Single(
                 viewContext.FormContext.FormData,
                 entry => entry.Key == SelectTagHelper.SelectedValuesFormDataKey);
-            Assert.NotNull(keyValuePair.Value);
-            var selectedValues = Assert.IsAssignableFrom<ICollection<string>>(keyValuePair.Value);
-            Assert.InRange(selectedValues.Count, 0, 1);
```

Comment:
not worth checking anymore.  `GetCurrentValues()` returns `null` in some cases where `GenerateSelect()` would set the `out selectedValues` argument to an empty collection.  (empty collection should now occur only if (a) a value is an empty `string`, (b) `allowMultiple` is `true` and a value is `null`, or (c) a value is a `string` that is neither a defined `enum` name nor an integer.)